===file:syntax/create_table_type.sql, id:5, sql:[INSERT INTO aly_test VALUES (2,2)]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:32:39.487847]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:10, sql:[INSERT INTO aly_test VALUES (1,01010)]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:32:39.819349]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:18, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-129)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:40.268527]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:19, sql:[INSERT INTO aly_test ( id,data) VALUES (2,128)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:40.347252]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:20, sql:[INSERT INTO aly_test ( id,data) VALUES (222,-129),(222,-128),(222,0),(222,127),(222,128)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:40.427045]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:29, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:41.017748]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:30, sql:[INSERT INTO aly_test ( id,data) VALUES (2,256)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:41.098043]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:31, sql:[INSERT INTO aly_test ( id,data) VALUES (111,-1),(112,0),(113,256)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:41.180985]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:40, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1),(2,-10000)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:41.767036]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:41, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,255),(114,256),(1112,2555),(115,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 4")
[13:32:41.850343]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:50, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1),(2,-10000)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:42.388889]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:51, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,255),(114,256),(1112,2555),(115,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 4")
[13:32:42.474735]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:62, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-32769)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:42.995273]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:63, sql:[INSERT INTO aly_test ( id,data) VALUES (2,32768)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:43.075938]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:64, sql:[INSERT INTO aly_test ( id,data) VALUES (222,-32769),(222,-32768),(222,0),(222,32767),(222,32768)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:43.155617]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:73, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:43.733839]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:74, sql:[INSERT INTO aly_test ( id,data) VALUES (2,65536)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:43.815975]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:75, sql:[INSERT INTO aly_test ( id,data) VALUES (111,-1),(112,0),(113,65536)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:43.898181]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:84, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1),(2,-65536)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:44.494655]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:85, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,65535),(114,655355),(115,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 4")
[13:32:44.577611]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:94, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1),(2,-65536)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:45.171995]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:95, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,65535),(114,655355),(115,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 4")
[13:32:45.254546]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:106, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-8388609)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:45.815495]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:108, sql:[INSERT INTO aly_test ( id,data) VALUES (222,-8388609),(222,-8388608),(222,0),(222,8388607),(222,8388608)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:45.977845]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:117, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:46.528782]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:118, sql:[INSERT INTO aly_test ( id,data) VALUES (2,16777216)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:46.609823]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:119, sql:[INSERT INTO aly_test ( id,data) VALUES (111,-1),(112,0),(113,16777216)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:46.695624]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:128, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1),(2,16777216)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:47.235523]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:129, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,16777215),(114,16777216),(115,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 4")
[13:32:47.319701]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:138, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1),(2,16777216)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:47.898581]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:139, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,16777215),(114,16777216),(115,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 4")
[13:32:47.984449]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:150, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-2147483649)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:48.536260]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:151, sql:[INSERT INTO aly_test ( id,data) VALUES (2,2147483648)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:48.617570]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:152, sql:[INSERT INTO aly_test ( id,data) VALUES (222,-2147483649),(222,-2147483648),(222,0),(222,2147483647),(222,2147483648)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:48.699310]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:161, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:49.265904]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:162, sql:[INSERT INTO aly_test ( id,data) VALUES (2,4294967296)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:49.345151]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:163, sql:[INSERT INTO aly_test ( id,data) VALUES (111,-1),(112,0),(113,4294967296)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:49.397375]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:172, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1),(2,-10000)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:49.963857]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:173, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,10000),(114,100000),(115,-1),(116,-10000)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 5")
[13:32:50.046652]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:182, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:50.583689]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:183, sql:[INSERT INTO aly_test ( id,data) VALUES (2,4294967296)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:50.663333]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:184, sql:[INSERT INTO aly_test ( id,data) VALUES (3,-1),(4,0),(5,4294967296)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:50.744730]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:185, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,10),(114,-1000),(115,-10000),(116,-100000),(117,10000),(118,4294967295)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 4")
[13:32:50.828919]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:189, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,10),(114,-1000),(115,-10000),(116,-100000),(117,10000),(118,4294967295)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 4")
[13:32:51.082996]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:195, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-2147483649)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:51.331237]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:196, sql:[INSERT INTO aly_test ( id,data) VALUES (2,2147483648)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:51.411985]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:197, sql:[INSERT INTO aly_test ( id,data) VALUES (222,-2147483649),(222,-2147483648),(222,0),(222,2147483647),(222,2147483648)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:51.492662]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:206, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:52.072004]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:207, sql:[INSERT INTO aly_test ( id,data) VALUES (2,4294967296)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:52.154439]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:208, sql:[INSERT INTO aly_test ( id,data) VALUES (111,-1),(112,0),(113,4294967296)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:52.205301]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:217, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1),(2,-10000)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:52.787957]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:218, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,10000),(114,100000),(115,-1),(116,-10000)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 5")
[13:32:52.864926]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:227, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:53.440979]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:228, sql:[INSERT INTO aly_test ( id,data) VALUES (2,4294967296)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:53.510912]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:229, sql:[INSERT INTO aly_test ( id,data) VALUES (3,-1),(4,0),(5,4294967296)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:53.586548]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:240, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-9223372036854775809)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:54.179921]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:241, sql:[INSERT INTO aly_test ( id,data) VALUES (2,9223372036854775808)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:54.260021]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:242, sql:[INSERT INTO aly_test ( id,data) VALUES (222,-9223372036854775809),(222,-9223372036854775808),(222,0),(222,9223372036854775807),(222,9223372036854775808)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:54.341804]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:251, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:54.927234]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:252, sql:[INSERT INTO aly_test ( id,data) VALUES (2,18446744073709551616)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:55.009727]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:253, sql:[INSERT INTO aly_test ( id,data) VALUES (111,-1),(112,0),(113,18446744073709551616)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:55.091990]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:262, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1),(2,18446744073709551615)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:55.717226]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:263, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,18446744073709551615),(115,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 4")
[13:32:55.799757]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:272, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1),(2,18446744073709551615)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:56.386183]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:273, sql:[INSERT INTO aly_test ( id,data) VALUES (111,0),(112,1),(113,18446744073709551615),(115,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 4")
[13:32:56.469689]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:285, sql:[INSERT INTO aly_test ( id,data) VALUES (2,1234567890123456789012345678901234567890.1234567890)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:57.073519]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:286, sql:[INSERT INTO aly_test ( id,data) VALUES (222,-1234567890123456789012345678901234567890.12345678901234),(222,-1234567890123456789012345678901234567890.1234567890123),(222,0),(222,1234567890123456789012345678901234567890.12345678901234),(222,-1234567890123456789012345678901234567890.12345678901234)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:57.157327]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:291, sql:[INSERT INTO aly_test ( id,data) VALUES (221,0.7777),(222,7777.777),(223,77777.77),(224,0),(225,7777777),(226,777777)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 3")
[13:32:57.475084]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:296, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:57.775115]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:301, sql:[INSERT INTO aly_test ( id,data) VALUES (2,18446744073709551616)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:58.146455]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:306, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:58.477576]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:311, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:58.813926]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:316, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:59.151280]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:321, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:32:59.487534]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:334, sql:[INSERT INTO aly_test ( id,data) VALUES (221,0.7777),(222,7777.777),(223,77777.77),(224,0),(225,7777777),(226,777777)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 3")
[13:33:00.229113]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:340, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:00.647231]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:345, sql:[INSERT INTO aly_test ( id,data) VALUES (2,18446744073709551616)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:01.009668]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:351, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:01.421222]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:357, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:01.830919]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:363, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:02.233191]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:369, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:02.614747]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:377, sql:[INSERT INTO aly_test ( id,data) VALUES (1111,-1234567890123456789012345678901234567890.1234567890123),(1112,0),(1113,1234567890123456789012345678901234567890.1234567890123)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:03.032134]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:382, sql:[INSERT INTO aly_test ( id,data) VALUES (221,0.7777),(222,7777.777),(223,77777.77),(224,0),(225,7777777),(226,777777)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 3")
[13:33:03.391836]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:388, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:03.773721]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:389, sql:[INSERT INTO aly_test ( id,data) VALUES (2,18446744073709551616),(3,0)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:03.854838]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:393, sql:[INSERT INTO aly_test ( id,data) VALUES (2,18446744073709551616)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:04.139208]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:399, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:04.520988]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:405, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:04.926019]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:411, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:05.335517]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:417, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:05.750248]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:425, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'a')]===
dble:[None]
mysql:[None]
mysql err:(1366, "Incorrect decimal value: 'a' for column 'data' at row 1")
[13:33:06.174235]dble err :(1366, "Incorrect decimal value: 'a' for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:426, sql:[INSERT INTO aly_test ( id,data) VALUES (2,1234567890123456789012345678901234567890123456789012345678901234567)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:06.254419]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:428, sql:[INSERT INTO aly_test ( id,data) VALUES (222,-1234567890123456789012345678901234567890.12345678901234),(222,-1234567890123456789012345678901234567890.1234567890123),(222,0),(222,1234567890123456789012345678901234567890.12345678901234),(222,-1234567890123456789012345678901234567890.12345678901234)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:06.390345]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:429, sql:[INSERT INTO aly_test ( id,data) VALUES (1111,-12345678901234567890123.4567890),(1112,0),(1113,12345678901234567890123.4567890)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:06.471991]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:434, sql:[INSERT INTO aly_test ( id,data) VALUES (221,0.7777),(222,7777.777),(223,77777.77),(224,0),(225,7777777),(226,777777)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 3")
[13:33:06.809679]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:440, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:07.195092]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:441, sql:[INSERT INTO aly_test ( id,data) VALUES (2,18446744073709551616),(3,0)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:07.278570]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:446, sql:[INSERT INTO aly_test ( id,data) VALUES (2,18446744073709551616)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:07.611127]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:452, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:08.057645]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:458, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:08.465013]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:464, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:08.897545]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:470, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:09.305788]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:478, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'a')]===
dble:[None]
mysql:[None]
mysql err:(1265, "Data truncated for column 'data' at row 1")
[13:33:09.738302]dble err :(1265, "Data truncated for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:487, sql:[INSERT INTO aly_test ( id,data) VALUES (221,0.7777),(222,7777.777),(223,77777.77),(224,0),(225,7777777),(226,777777)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 3")
[13:33:10.346542]dble err :(1264, "Out of range value for column 'data' at row 2")
===file:syntax/create_table_type.sql, id:493, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:10.728758]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:499, sql:[INSERT INTO aly_test ( id,data) VALUES (2,18446744073709551616)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:11.144333]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:505, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:11.558719]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:511, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:11.963303]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:517, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:12.370950]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:523, sql:[INSERT INTO aly_test ( id,data) VALUES (1,-1)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:12.784229]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:532, sql:[INSERT INTO aly_test ( id,data) VALUES (2,10)]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:13.289918]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:534, sql:[INSERT INTO aly_test ( id,data) VALUES (3,'aaaaaaaaaaaa,aaaaaaaaaaaa_aaaaaaaaa%%aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa*aa')]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:13.454291]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:540, sql:[INSERT INTO aly_test ( id,data) VALUES (2,10)]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:13.895592]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:546, sql:[INSERT INTO aly_test ( id,data) VALUES (1,123456)]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:14.309415]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:547, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'abcdef')]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:14.389965]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:564, sql:[INSERT INTO aly_test ( id,data) VALUES (1,123456)]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:15.388462]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:565, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'abcdef')]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:15.468901]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:566, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'abcde')]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:15.549691]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:570, sql:[INSERT INTO aly_test ( id,data) VALUES (1,123456)]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:15.831707]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:571, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'abcdef')]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:15.911107]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:572, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'abcde')]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:15.992198]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:577, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'aaaaaaaaaaaaaaaaaaaaaaaaabcdefg')]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:16.354396]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:583, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'aaaaaaaaaaaaaaaaaaaaaaaaabcdefg')]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:16.760537]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:591, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'201411-11')]===
dble:[None]
mysql:[None]
mysql err:(1292, "Incorrect date value: '201411-11' for column 'data' at row 1")
[13:33:17.143133]dble err :(1292, "Incorrect date value: '201411-11' for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:596, sql:[INSERT INTO aly_test ( id,data) VALUES (6,'aaaa-01-01')]===
dble:[None]
mysql:[None]
mysql err:(1292, "Incorrect date value: 'aaaa-01-01' for column 'data' at row 1")
[13:33:17.494695]dble err :(1292, "Incorrect date value: 'aaaa-01-01' for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:599, sql:[INSERT INTO aly_test ( id,data) VALUES (9,'10000-01-01')]===
dble:[None]
mysql:[None]
mysql err:(1292, "Incorrect date value: '10000-01-01' for column 'data' at row 1")
[13:33:17.707539]dble err :(1292, "Incorrect date value: '10000-01-01' for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:600, sql:[INSERT INTO aly_test ( id,data) VALUES (10,'9999-01-100')]===
dble:[None]
mysql:[None]
mysql err:(1292, "Incorrect date value: '9999-01-100' for column 'data' at row 1")
[13:33:17.788427]dble err :(1292, "Incorrect date value: '9999-01-100' for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:603, sql:[INSERT INTO aly_test( id,date) VALUES (13,'20170201')]===
dble:[None]
mysql:[None]
mysql err:(1054, "Unknown column 'date' in 'field list'")
[13:33:17.999605]dble err :(1054, "Unknown column 'date' in 'field list'")
===file:syntax/create_table_type.sql, id:604, sql:[INSERT INTO aly_test( id,date) VALUES (14,'170202')]===
dble:[None]
mysql:[None]
mysql err:(1054, "Unknown column 'date' in 'field list'")
[13:33:18.079119]dble err :(1054, "Unknown column 'date' in 'field list'")
===file:syntax/create_table_type.sql, id:611, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'201411-11')]===
dble:[None]
mysql:[None]
mysql err:(1292, "Incorrect time value: '201411-11' for column 'data' at row 1")
[13:33:18.430938]dble err :(1292, "Incorrect time value: '201411-11' for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:627, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'1970-01-01 00:00:01'),(2,'2038-01-19 03:14:07')]===
dble:[None]
mysql:[None]
mysql err:(1292, "Incorrect datetime value: '1970-01-01 00:00:01' for column 'data' at row 1")
[13:33:19.339442]dble err :(1292, "Incorrect datetime value: '1970-01-01 00:00:01' for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:646, sql:[INSERT INTO aly_test ( id,data) VALUES (1,1900)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:20.360254]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:647, sql:[INSERT INTO aly_test ( id,data) VALUES (2,2156)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 1")
[13:33:20.439884]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:648, sql:[INSERT INTO aly_test ( id,data) VALUES (3,'2038-01-19 03:14:07')]===
dble:[None]
mysql:[None]
mysql err:(1265, "Data truncated for column 'data' at row 1")
[13:33:20.520770]dble err :(1265, "Data truncated for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:649, sql:[INSERT INTO aly_test ( id,data) VALUES (4,1901),(5,2156),(6,0000)]===
dble:[None]
mysql:[None]
mysql err:(1264, "Out of range value for column 'data' at row 2")
[13:33:20.605573]dble err :(1264, "Out of range value for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:661, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'abcdef')]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:21.229662]dble err :(1406, "Data too long for column 'data' at row 1")
===file:syntax/create_table_type.sql, id:669, sql:[INSERT INTO aly_test ( id,data) VALUES (1,'abcdef')]===
dble:[None]
mysql:[None]
mysql err:(1406, "Data too long for column 'data' at row 1")
[13:33:21.667317]dble err :(1406, "Data too long for column 'data' at row 1")
